src/NuGet.Clients/NuGet.Tools/NuGetPackage.cs

```diff
@@ -58,7 +58,7 @@ namespace NuGetVSExtension
     [ProvideOptionPage(typeof(StubGeneralOptionsPage), "NuGet Package Manager", "GeneralStub", 113, 115, true, IsInUnifiedSettings = false, Sort = 0,
         // Only show the stub legacy settings General page by using a visibility context indicating when the unified settings feature is enabled.
         VisibilityCmdUIContexts = "13b56f17-0b98-4a51-a9a0-9767d84796da")]
-    [ProvideOptionPage(typeof(ConfigurationFilesOptionsPage), "NuGet Package Manager", "Configuration Files", 113, 117, true, Sort = 1)]
```

Comment:
Keep the `Sort` property so it continues to sort properly in the old Options dialog.